packages/@aws-cdk/cfnspec/lib/augmentations/AWS_EC2_VPNConnection.json

```diff
@@ -0,0 +1,27 @@
+{
+  "overrides": {
```

Comment:
I wouldn't call these "overrides". We should think about them simply as configuration of the augmentation system that has some default values, so you don't have to supply them but you can, and I would avoid attaching any heuristics to them (i.e. "Base" suffix).

So, maybe:

```json
{
  "baseClass": "VpnConnectionBase",
  "baseClassFile": "vpn",
  "interface": "IVpnConnection"
}
```

Reply:
OK, will rename these. The current implementation doesn't attach anything to these overrides.

Reply:
It does:

- `baseClass` is the resource name + "Base" suffix
- `baseClassFile` is the `l2-base`
- `interface` is `I` + resource name

Reply:
Not sure I'm following here.

```ts
const baseClassName = (overrides && overrides.class) || l2ClassName + 'Base';
const interfaceName = (overrides && overrides.interface) || 'I' + l2ClassName;
const baseClassModule = `./${(overrides && overrides.module) || `${kebabL2ClassName}-base`}`;
```

From above, if there's an override it's taken as is, no?

Regarding the names in the JSON file:
- `baseClass` or `class`? for the VpnConnection it's not a base class because there's no VpnConnectionBase in this case
- `...file` or `...module`?

Reply:
Oh, yes, you are right. I guess `class` makes sense. I would go with `file` and not `module` because we mostly (in our project) "module" is interchangeable with "package" (or "npm module").